### ISSUE TITLE
/opt folder

### ISSUE THREAD

**Issue by mrdvt92** [id/kafka-el7-rpm#21]
I think if the goal of this RPM is to be integrated into Fedora/RedHat/CentOS the path should be /usr/share/kafka not /opt.

**Comment by id** [comment]
I agree. Unfortunately don't have time to work on this project anymore.
